subiquity/server/controllers/install.py

```diff
@@ -166,6 +166,20 @@ def supports_apt(self) -> bool:
             and self.model.source.current.variant != "core"
         )
 
+    def write_autoinstall_config(self) -> None:
+        autoinstall_path = os.path.join(
+            self.app.root, "var/log/installer/autoinstall-user-data"
+        )
+        autoinstall_config = (
+            "#cloud-config\n"
+            "# See the auto install documentation at:"
+            "# https://canonical-subiquity.readthedocs-hosted.com/en/latest/reference/autoinstall-reference.html\n"  # noqa: E501
```

Comment:
Not willing to block your PR on this but do we expect this URL to be valid going forward?

Reply:
For now, but I'm pretty sure the `canonical-subiquity.readthedocs-host.com` part will change at some point as the docs evolve. I wanted to put this somewhere intelligent to only have to change it in one spot, but hadn't come up with anything nice.

Reply:
OK, I don't think it's a big deal.